content/aips/PROTOCOL_UPGRADE_3_0_1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Upgrade Aave V3 pools to Aave V3.0.1
+author: BGD Labs (@bgdlabs)
+shortDescription: Upgrade various contracts to V3.0.1 to align Aave Deployments
+discussions: TBA
+created: 2023-02-10
+---
+
+## Simple Summary
+
+This proposal upgrades the Aave v3 instances on Polygon, Avalanche, Optimism and Arbitrum to the v3.0.1 version, already running on the new Aave v3 Ethereum pool.
+If this proposal succeeds the success will be seen as signal to also upgrade the V3 pools on Fantom, and Harmony via a delegatecall from the guardians.
```

Comment:
```suggestion
If this proposal succeeds, it will also authorize the upgrade of the V3 pools on Avalanche, Fantom, and Harmony via a delegatecall by the Aave Guardian.
```